hooks/post_gen_project.py

```diff
@@ -0,0 +1,6 @@
+import subprocess
+
+subprocess.call(['git', 'init'])
+subprocess.call(['npm', 'install'])
+subprocess.call(['git', 'add', '*'])
```

Comment:
[sand] Would the following be better?

```suggestion
subprocess.call(['git', 'add', '.'])
```

Globbing seems to be expanded by `git` itself, and `*` seems to include dotfiles, so this might behave the same.